.github/workflows/ci.yml

```
@@ -22,6 +22,4 @@ jobs:
       - uses: julia-actions/julia-buildpkg@latest
       - uses: julia-actions/julia-runtest@latest
       - uses: julia-actions/julia-uploadcodecov@latest
```

```suggestion
      - uses: codecov/codecov-action@v1
```
sorry we have to also exchange the action as it seems.